app/subscriber/src/components/sentiment/Sentiment.tsx

```diff
@@ -3,28 +3,28 @@ import { Show } from 'tno-core';
 
 import * as styled from './styled';
 
-export interface IDetermineToneIconProps {
+export interface ISentimentProps {
```

Comment:
@jdtoombs I've moved and renamed this component.  We probably should move it to `tno-core`